app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb

```diff
@@ -1 +1,35 @@
-confirmation
+<div class="govuk-box-highlight">
+  <h1 class="bold-large"> <%= page_title t('.heading') %></h1>
+  <p>
+    <%= t('.para_text') %>
+    <br />
+    <strong class="heading-medium"><%= form.enrollment.reference_number %></strong>
+  </p>
+</div>
+
+<p class="text"><%= t ".email_sent", email: form.email %></p>
+
+<h2 class="heading-medium">
+  <%= t('.heading_h2') %>
+</h2>
+
+<p class="text"><%= t('.para_text2') %></p>
+<p class="text"><%= t('.para_text3') %></p>
+<p class="text"><%= t('.para_text4') %></p>
+<p class="text"><%= t('.para_text5') %>
```

Comment:
Sorry - working with multiple languages is where this issue has caught me out in the past, and is the simplest example to show what the problem is. I realise that a second language is unlikely to be needed, but can't help seeing this as something that can be done better.

Feel free to ignore.